docker/Dockerfile.nvidia

```diff
@@ -124,62 +106,19 @@ RUN if [ "x$MPI_VER" = "x4" ]; then \
         rm -f  $HPCSDK_HOME/comm_libs/mpi && \
         ln -sf $HPCSDK_HOME/comm_libs/hpcx/latest/ompi \
                $HPCSDK_HOME/comm_libs/mpi ; \
-    fi;  \
-    /venv/bin/pip install --no-cache-dir -r /app/requirements-mpi.txt && \
-    rm -rf ~/.cache/pip
+    fi;
```

Comment:
what is this?

Reply:
I'm not 100% sure. It is in master. I think it is meant to use different versions of MPI as per a docker build arg.

Reply:
ok